src/client/containers/RegistrationPage/StudentRegistrationPage/StudentRegistrationPage.component.js

```diff
@@ -52,10 +53,7 @@ export const StudentRegistrationPage = () => {
       <div className="student-registration-button">
         <Button>Submit</Button>
       </div>
-      <p className="student-registration-info-text">
-        If you already do not have a slack id then please make one as it is
-        mandatory to have a slack id to connect with mentors and other students.
-      </p>
+      <HelpText klass="student-registration-info-text" />
```

Comment:
Keep the text here.

Reply:
Hey Toni, I want to enquire why you asked me to keep the text here, while the component is intended to replace the custom text version.